e2e/tests/strategy/recurring/edit.ts

```diff
@@ -24,6 +29,30 @@ export const editPriceStrategyTest = (testCase: CreateStrategyTestCase) => {
       'Your strategy was successfully updated.'
     );
 
-    // TODO Assert new prices from tooltips
+    const myStrategies = new MyStrategyDriver(page);
+    const strategyEdited = await myStrategies.getStrategy(1);
+
+    // Check range
+    const [buySetting, sellSetting] = getRecurringSettings(testCase);
+
+    const buyTooltip = await strategyEdited.priceTooltip('buy');
+    if (buySetting === 'limit') {
+      await expect(buyTooltip.price()).toHaveText(buy.min);
+      await expect(buyTooltip.price()).toHaveText(buy.max);
```

Comment:
You don't need to verify twice. Here you are checking if buy.min === buy.max which is outside the scope of E2E
```suggestion
      await expect(buyTooltip.price()).toHaveText(buy.min);
```

Reply:
agreed